src/Query/User/PackageQuery/DbalPackageQuery.php

```diff
@@ -49,13 +50,16 @@ public function findAll(string $organizationId, int $limit = 20, int $offset = 0
                 last_scan_result
             FROM organization_package
             WHERE organization_id = :organization_id
+            AND (name ILIKE :namesearch OR description ILIKE :descsearch)
             GROUP BY id
             ORDER BY name ASC
             LIMIT :limit OFFSET :offset', [
                 ':organization_id' => $organizationId,
-                ':limit' => $limit,
-                ':offset' => $offset,
-            ]));
+                ':limit' => $filter->getLimit(),
+                ':offset' => $filter->getOffset(),
+                ':namesearch' => '%'.$filter->getSearchTerm().'%',
+                ':descsearch' => '%'.$filter->getSearchTerm().'%',
```

Comment:
`namesearch` and `descsearch` can be replaced with one variable, there is no need for two identical params.
Also, we need to split this sql string to parts, for example:
```php
        $params = [];
        $sql = 'SELECT
                id,
        // ...
                last_scan_result
            FROM organization_package
            WHERE organization_id = :organization_id';
        
        if($filter->hasSerachTerm()) {
            $sql .= ' AND (name ILIKE :term OR description ILIKE :term)';
            $params[':term'] = $filter->serachTerm();
        }
        
        $sql .= ' GROUP BY id
            ORDER BY name ASC
            LIMIT :limit OFFSET :offset';
```